scss/_layout_grid.scss

```diff
@@ -11,3 +11,14 @@ $shelves-column-name: $grid-col-name;
 @import
 'grid/shelves',
 'grid/shelves-grid';
+
+/* Box-sizing FTW!
+https://css-tricks.com/inheriting-box-sizing-probably-slightly-better-best-practice/
+*/
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
```

Comment:
Shouldn't any `*` rules and rules applied to elements directly (especially `html`) be in a `_base` file?